src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -21,46 +31,66 @@ import topology2 from 'images/topology-2.png'
 
 import topology3 from 'images/topology-3.png'
 
-Our [incident intelligence](/docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-incident-intelligence) uses logic to correlate your [incidents](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence#issue-feed). These logical operations are called **decisions**. We have built-in decisions, and you can create and customize your own on the incident intelligence **Decisions** UI page. The more optimally you configure decisions, the better we will group and correlate your incident events, resulting in reduced noise and increased context for on-call teams.
+With applied intelligence's correlation logic, alerts and events from any source are automatically grouped into related issues.  As events come into your system they are instantly correlated based on time, alert context, and relationship data. If multiple events are related then our correlation logic will funnel the related incidents into a single, comprehensive [issue](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence#issue-feed). This ensures that your team is alerted to relevant issues as soon as possible while also reducing distracting and redundant alerts.
+
+We call this correlation logic **decisions**. We have built-in decisions but you can also create and customize your own on the decisions page **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Decisions**. The more you configure your decisions to best suit your needs,  the better New Relic can correlate your incidents, reduce noise and provide increased context for on-call teams.
```

Comment:
the pathing needs to be introduced and not just part of the sentence. for example, "on the decisions page. To find the decisions page, go to:..."

extra space after comma

needs serial comma after 'reduce noise'

Reply:
done! thanks